guides/hotwire-turbo-micronaut-chat/complete/java/src/main/resources/views/layout.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html th:fragment="layout (script, content)" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>Chat</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
```

Comment:
```suggestion
    <meta name="viewport" content="width=device-width,initial-scale=1" />
```

Reply:
it is valid in HTML 5 not to close the elements.